contentrepo/settings/base.py

```diff
@@ -3,8 +3,10 @@
 DEBUG = True
 PROJECT_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 BASE_DIR = os.path.dirname(PROJECT_DIR)
-SECRET_KEY = 'vg%%_7^bt(68qg50vgp^*f^&_(f)36k1vnyg%49n8*0s86hxj6'
-
+SECRET_KEY = os.environ.get(
+    'SECRET_KEY', 'vg%%_7^bt(68qg50vgp^*f^&_(f)36k1vnyg%49n8*0s86hxj6')
```

Comment:
I think it's better to not have a default Secret_key. that way if we forget to put one in the Env vars the deploy will fail, rather than the application quietly running with a vulnerable key.

Reply:
okay